src/game.py

```diff
@@ -8,6 +8,38 @@ def get_welcome_message(self):
     def initialize_board(self):
         return f"{self.board[0]} | {self.board[1]} | {self.board[2]}\n--+--+--\n{self.board[3]} | {self.board[4]} | {self.board[5]}\n--+--+--\n{self.board[6]} | {self.board[7]} | {self.board[8]}"
 
+    def count_plays(self):
+        play_count = self.board.count("X") + self.board.count("O")
+        return play_count
+
+    def get_next_player(self, number):
+        if number == 0:
+            return "X"
+        elif number % 2 == 0:
+            return "X"
+        else:
+            return "O"
+
+    def get_prompt(self, play_count):
+        next_player = self.get_next_player(play_count)
+        if play_count == 9:
```

Comment:
Similarly to the above question, what if we changed the size of the board?

Reply:
Fixed this with: `len(self.board)` instead of hard-coding 9 so if we change the length of the board, it won't break.